NanoAnalysis/python/modules/jetJERC.py

```diff
@@ -0,0 +1,73 @@
+import os
+
+def getJetCorrected(era, tag, is_mc, overwritePt=True) :
+    from PhysicsTools.NATModules.modules.jetCorr import jetJERC
+
+    if era != 2022:
+        raise ValueError("getJetCorrected: Era", era, "not supported")
+
+    if is_mc :
+        if tag == "pre_EE":
```

Comment:
can you set 
`      if "pre_EE" in tag :`
(as done in eg [eleScaleResProducer.py](https://github.com/CJLST/ZZAnalysis/blob/Run3/NanoAnalysis/python/modules/eleScaleResProducer.py#L16)? The reason is that we may add more strings in data_tag, e.g. we may have at some point an "ultra-legacy" rereco to distinguish from this one.

Reply:
Done